lib/internal/test_runner/utils.js

```diff
@@ -242,7 +242,7 @@ function parseCommandLine() {
 
   if (isTestRunner) {
     isolation = getOptionValue('--test-isolation');
-    timeout = getOptionValue('--test-timeout') || Infinity;
+    timeoutPerTest = getOptionValue('--test-timeout') || Infinity;
```

Comment:
This and line 274 are the same, so they can be hoisted out of this `if`, similar to the variables on lines 193-198.

Reply:
Good catch 👍 done